### ISSUE TITLE
Error While installing Mod

### ISSUE THREAD

**Issue by Genaeve** [TanninOne/modorganizer#838]
When installing a mod that uses Material folder, rather than textures, sound, etc, I got the message "No Game Data on Top Level."
I uninstalled, reinstalled, did a clean install, a few times. Same result.
Clearly, there was Game Data on Top Level. I've dropped a pic comparing the 2 so you can see. 

![nodata](https://cloud.githubusercontent.com/assets/20137342/16358389/9accf8c0-3ac6-11e6-8b66-1a5379c7e867.jpg)

~Gena

**Comment by Zakrael** [comment]
Same issue here, mod was http://www.nexusmods.com/fallout4/mods/1939/? if it's helpful.

The mod itself still installed correctly, though, it just seems to be an issue with MO giving a false error. After selecting the top-level data folder I hit OK and ignored the warning, and the mod worked as advertised. 

![nogamedata](https://cloud.githubusercontent.com/assets/20270196/16547039/96baf9ee-4156-11e6-8f31-8a0fb797ce40.png)

**Comment by TanninOne** [comment]
Yes, materials is not currently a recognized top-level dir, next version fixes that
